astropy/io/votable/tests/test_table.py

```diff
@@ -518,6 +518,174 @@ def test_validate_output_valid():
     assert validate_out is True
 
 
+def test_binary2_single_bounded_char_array():
+    """
+    Test that variable length char arrays with a max length
+    (arraysize="128*") are read correctly in BINARY2 format.
+    """
+    votable = parse(get_pkg_data_filename("data/binary2_variable_length_char.xml"))
```

Comment:
It is choking here. I do not see this file added in this PR. Are you sure this is the correct filename?

Reply:
Yea I must have not added that data file in the original commit, I've added it now thanks